tools/realsense-viewer/CMakeLists.txt

```diff
@@ -109,12 +109,12 @@ endif()
 
 # config-ui
 if(WIN32)
-    add_executable(realsense-viewer WIN32
+    add_executable(realsense-viewer 
         ${RS_VIEWER_CPP}
         ${ELPP_FILES}
         ${CMAKE_CURRENT_SOURCE_DIR}/res/resource.h
         ${CMAKE_CURRENT_SOURCE_DIR}/res/realsense-viewer.rc
-        ../../common/windows-app-bootstrap.cpp)
+        )
```

Comment:
Is it aligned with @dorodnic ?

Reply:
This is weird, I don't know where this is from...

Reply:
Seems this makes the viewer open a console window. So this should NOT go in. I will revert it.